### ISSUE TITLE
3a cannot find SCEN_MAP symbol in output GDX

### ISSUE THREAD

**Issue by acjbrouwer** [iiasa/GLOBIOM-G4M-link#3]
When running link commit 496a295 with the therein-referenced submodules recursively cloned and the current GLOBIOM Trunk, chunk 3a throws error
```
Error in rgdx(gdxName, list(name = symName, compress = compress, ts = ts, : 
GDX file E:/work/GLOBIOM/links/GLOBIOM-G4M-link/GLOBIOM/Model/gdx/output_test_Link_74.000000.gdx contains no symbol named 'SCEN_MAP'
```
from `helper_functions.R` -> `get_mapping()` -> `rgdx.set()`.

**Comment by acjbrouwer** [comment]
Somehow my `<link root>/GLOBIOM/Model/outputs_limpopo.gms` is lacking the SCEN_MAP entry present in the latest Trunk. Will just flush everything and re-do the clone, checkout, pre-compilation, and model stages 1-4.

**Comment by acjbrouwer** [comment]
The issue was that `checkout.bat` referenced an outdated Trunk commit. Fixed in 200da28fb639ce009f9075d9c1affaedd1857c35.
